e2e/log.go

```diff
@@ -0,0 +1,55 @@
+package e2e
+
+import (
+	"log"
+	"os"
+	"path/filepath"
+
+	collector "github.com/coreos/ktestutil/log-collector"
+)
+
+var cr *collector.Collector
+
+func startLogging(keypath, dstDir string) error {
+	cr = collector.New(&collector.Config{
+		K8sClient:     client,
+		Namespace:     namespace,
+		RemoteKeyFile: keypath,
+	})
+	if err := cr.Start(); err != nil {
+		return err
+	}
+
+	dirAbsPath, err := filepath.Abs(dstDir)
+	if err != nil {
+		return err
+	}
+	os.Mkdir(dirAbsPath, 0777)
+	if err := cr.SetOutputToLocal(dirAbsPath); err != nil {
+		return err
+	}
+	return nil
+}
+
+func collectLogs() error {
+	pods, err := cr.CollectPodLogs("*")
+	if err != nil {
+		return err
+	}
+	log.Println("pods", pods)
```

Comment:
Is there a better way to print this, or do we need this?
The output I saw is not very useful imo:
```
[122504.503025] golang[5]: rc/github.com/kubernetes-incubator/bootkube/e2e/logs/container.fluentd-worker-jjqdz.kube-calico-vw4cv_kube-system_kube-calico.b552914be74f0a3c5738a83dce3f9aba5.log /go/src/github.com/kubernetes-incubator/bootkube/e2e/logs/container.ec2.pod-checkpointer-1kzkd-ip-10-63-54-223.20170622.log /go/src/github.com/kubernetes-incubator/bootkube/e2e/logs/container.fluentd-worker-jjqdz.kube-scheduler-1873228005-gxpkz_kube-system_kube-scheduler.20170622.log /go/src/github.com/kubernetes-incubator/bootkube/e2e/logs/container.fluentd-worker-jjqdz.kube-calico-vw4cv_kube-system_install-cni.20170622.log /go/src/github.com/kubernetes-incubator/bootkube/e2e/logs/container.fluentd-worker-jjqdz.kube-controller-manager-762207937-g9kd5_kube-system_kube-controller-manager.b552914c3368d9c48e29279f5e869fae6.log /go/src/github.com/kubernetes-incubator/bootkube/e2e/logs/container.ec2.pod-checkpointer-0z7kg-ip-10-233-85-248.b5529110f9188d9c50ec010b59fe97292.log /go/src/github.com/kubernetes-incubator/bootkube/e2e/logs/container.fluentd-worker-63hrb.kube-calico-jrllz_kube-system_install-cni.20170622.log /go/src/github.com/kubernetes-incubator/bootkube/e2e/logs/container.fluentd-worker-jjqdz.pod-checkpointer-kgzzg_kube-system_pod-checkpointer.20170622.log /go/src/github.com/kubernetes-incubator/bootkube/e2e/logs/container.fluentd-worker-5cmp8.kube-apiserver-pmmv7_kube-system_kube-apiserver.20170622.log /go/src/github.com/kubernetes-incubator/bootkube/e2e/logs/container.fluentd-worker-jjqdz.kube-flannel-3kbm3_kube-system_install-cni.20170622.log /go/src/github.com/kubernetes-incubator/bootkube/e2e/logs/container.fluentd-worker-jjqdz.kube-apiserver-g9dtf_kube-system_kube-apiserver.20170622.log /go/src/github.com/kubernetes-incubator/bootkube/e2e/logs/container.fluentd-worker-jjqdz.kube-proxy-9mtjs_kube-system_kube-proxy.20170622.log /go/src/github.com/kubernetes-incubator/bootkube/e2e/logs/container.fluentd-worker-63hrb.pod-checkpointer-g91d5_kube-system_pod-checkpointer.20170622.log /go/src/github.com/kubernetes-incubator/bootkube/e2e/logs/conta
```